src/api/app/components/diff_list_component.rb

```diff
@@ -18,13 +18,13 @@ def initialize(diff_list:, view_id: nil, commentable: nil, commented_lines: {},
   # We expand the diff if the changeset:
   # it's not for a deletion
   # and it is not a directory
-  # and it's a _patchinfo, *.spec or *.changes file
+  # and it's a _patchinfo, a Dockerfile, *.spec or *.changes file
   # or someone commented on the diff
   def expand?(filename, state, file_index)
     return true if
       state != 'deleted' &&
       filename.exclude?('/') &&
-      (filename == '_patchinfo' || filename.ends_with?('.spec', '.changes'))
+      (filename == '_patchinfo' || filename == 'Dockerfile' || filename.ends_with?('.spec', '.changes'))
```

Comment:
This means we handle only the case when the basic convention is used by naming the file `Dockerfile`. Maybe catching also the files **with** the word `Dockerfile` included can be intercepted. AFAIK it's a common pattern to name `Dockerfile.frontend` `Dockerfile.dev` or also `backend.docker` `test.docker` etc.

Reply:
Done. Please take a look again.